node/src/database/db_migrations.rs

```diff
@@ -250,6 +271,75 @@ impl DatabaseMigration for Migrate_2_to_3 {
     }
 }
 
+#[derive(Debug)]
+#[allow(non_camel_case_types)]
+struct Migrate_4_to_5;
+
+impl DatabaseMigration for Migrate_4_to_5 {
+    fn migrate<'a>(
+        &self,
+        declaration_utils: Box<dyn MigDeclarationUtilities + 'a>,
+    ) -> rusqlite::Result<()> {
+        let mut select_statement = declaration_utils
+            .transaction()
+            .prepare("SELECT pending_payment_transaction FROM payable WHERE pending_payment_transaction IS NOT NULL")?;
+        let unresolved_pending_transactions: Vec<String> = select_statement
+            .query_map([], |row| {
+                Ok(row
+                    .get::<usize, String>(0)
+                    .expect("select statement was badly prepared"))
+            })?
+            .flatten()
+            .collect();
+        if !unresolved_pending_transactions.is_empty() {
+            warning!(declaration_utils.logger(),"Migration from 4 to 5: database belonging to the chain '{}'; \
+             we discovered possibly abandoned transactions that are said yet to be pending, these are: '{}'; \
+              continuing",declaration_utils.external_parameters().chain.rec().literal_identifier,unresolved_pending_transactions.join("', '") )
+        } else {
+            debug!(
+                declaration_utils.logger(),
+                "Migration from 4 to 5: no previous pending transactions found; continuing"
+            )
+        };
+        let statement_1 = "ALTER TABLE payable RENAME TO _payable_old";
```

Comment:
This should work, but if you want to save a little complexity, you could keep the old `payable` table in place and just use a couple of ALTER TABLE statements to drop the old column and add a new column. That way you wouldn't have to copy any data.